mne/time_frequency/tfr.py

```diff
@@ -281,8 +281,8 @@ def cwt_morlet(X, sfreq, freqs, use_fft=True, n_cycles=7.0, zero_mean=False,
     """
     mode = 'same'
     # mode = "valid"
-    n_signals, n_times = X.shape
     decim = int(decim)
```

Comment:
I'm actually wondering whether we should have this `int(decim)` in the first place. It would crash on the next line if it wasn't an int, and we don't really want people to pass `float` that will get rounded.